CHANGELOG.md

```diff
@@ -8,6 +8,46 @@
 > requests](https://github.com/fluxcd/toolkit/discussions) for Flux v2
 > and Toolkit components.
 
+## 1.21.2 (2021-02-09)
+
+This patch release surfaces a notice from https://docs.fluxcd.io/en/latest/
+declaring that "Flux v1 is in maintenance mode" into the Flux v1 stable docs,
+as well as pointing users from as many places as possible to the new Flux v2.
+
+### Fixes
+
+- Update AWS SDK and SOPS [fluxcd/flux#3401][]
+- Set Rollout strategy type: Recreate (Fixes for chart 1.6.2)
+  [fluxcd/flux#3325][]
+
+### Maintenance and documentation
+
+- Release chart 1.6.1 [fluxcd/flux#3416][]
```

Comment:
This is not related to the release of Flux and can be omitted.

Reply:
Thanks. Dropped this from cef604e5 which I've just force-pushed over, it has this.